src/transformers/modeling_t5.py

```diff
@@ -780,6 +897,17 @@ def set_input_embeddings(self, new_embeddings):
         self.encoder.set_input_embeddings(new_embeddings)
         self.decoder.set_input_embeddings(new_embeddings)
 
+    def set_output_past(self, do_output_past: bool):
+        self.config.output_past = do_output_past
+        self.decoder.output_past = do_output_past
```

Comment:
You also need to set output_past in the blocks and attention layers:
`for block in self.decoder.block:
            block.output_past = do_output_past
            block.layer[0].SelfAttention.output_past = do_output_past
            block.layer[1].EncDecAttention.output_past = do_output_past`